### PR TITLE
(IAC-1446) - Removal of Inappropriate Terminology

### PR DESCRIPTION
**Pull request by david22swan**
Removal of any offensive or inappropriate terminology from the code base.

Related PR: https://github.com/puppetlabs/ci-job-configs/pull/7690

### CODE REVIEW

File: configs/components/puppet-specifications.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 component "puppet-specifications" do |pkg, settings, platform|
-  pkg.ref "master"
+  pkg.ref "main"
```
Comment (DavidS) [inline configs/components/puppet-specifications.rb:2]: This one is not updated yet, and it is not in our team's scope.

File: README.md [hunk 2]
```diff
@@ -8,8 +8,8 @@ This repo contains all the packaging scripts to build the native PDK packages ac
...
 
-1. Choose the upstream pdk SHA that you want to release. Optionally, follow the pdk's [release process](https://github.com/puppetlabs/pdk/blob/master/CONTRIBUTING.md#release-process) to create a public gem release. **You can ignore this step if you want to take the latest commit on `master` of the [pdk](https://github.com/puppetlabs/pdk/blob/master) and [pdk-templates](https://github.com/puppetlabs/pdk-templates/blob/master) repos.**
-2. PDK merges to master are automatically promoted into pdk-vanagon with [this jenkins job](https://jenkins-master-prod-1.delivery.puppetlabs.net/view/PDK/job/platform_pdk-vanagon-promotion_pdk-van-promote_master/).
+1. Choose the upstream pdk SHA that you want to release. Optionally, follow the pdk's [release process](https://github.com/puppetlabs/pdk/blob/main/CONTRIBUTING.md#release-process) to create a public gem release. **You can ignore this step if you want to take the latest commit on `main` of the [pdk](https://github.com/puppetlabs/pdk/blob/main) and [pdk-templates](https://github.com/puppetlabs/pdk-templates/blob/main) repos.**
```
Comment (DavidS) [inline README.md:11]: both of the `/blob/main` links go to an error page. Please fix to link to the repo main page instead.

File: configs/components/puppet-specifications.rb (hunk 1 above)
Comment (david22swan) [reply configs/components/puppet-specifications.rb]: HAve reverted

File: README.md (hunk 2 above)
Comment (david22swan) [reply README.md]: They now point to the default branch just

**Pull request merged**